frontend/projects/ui/src/app/modals/generic-input/generic-input.component.module.ts

```diff
@@ -5,6 +5,7 @@ import { IonicModule } from '@ionic/angular'
 import { RouterModule } from '@angular/router'
 import { SharedPipesModule } from '@start9labs/shared'
 import { FormsModule } from '@angular/forms'
+import { MaskPipeModule } from 'src/app/pipes/mask/mask.module'
```

Comment:
I'm not sure you need to import this at all, since you are not using the pipe in html. Could be wrong. Try it.